src/main/kotlin/com/zepben/ewb/database/sqlite/cim/CimDatabaseReader.kt

```diff
@@ -59,14 +59,14 @@ abstract class CimDatabaseReader<TTables : CimDatabaseTables, TService : BaseSer
     /**
      * Read the database.
      */
-    fun read(service: TService): Boolean =
+    fun read(service: TService, afterServiceReadFunction: Boolean? = true): Boolean =
```

Comment:
todo: Can you please update the docstring for the public function to include the variables (more of that code improvement stuff)